src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -2622,6 +2622,68 @@ public void ParseReferences()
             // TODO: multiple files, quotes, etc.
         }
 
+        [Fact]
+        public void ParseReferencesAlias()
+        {
+            Assert.True(parseRef(@"/r:a=util.dll") is
```

Comment:
Why not `Assert.Equal`? Seems like that would give better error messages in case it fails.

Reply:
Couldn't find a good way to use pattern matching and `Assert.Equal`.

Reply:
Hand coded a custom `assert` method for this test.